src/command.ts

```diff
@@ -0,0 +1,44 @@
+import { isFunction, isString } from 'util'
+import { Command } from './types'
+
+const commandParse: RegExp = /^(?:(\w+)\.)?(?:(\w+):)?(\w+)$/
+
+export function parseCommand (queueCommand: any): Command {
+  const cmd: any = {}
+  if ('function' === typeof queueCommand[0]) {
+    cmd.callback = queueCommand[0]
+  } else {
+    const command: RegExpExecArray | null = commandParse.exec(queueCommand[0])
+    if (command !== null && 4 === command.length) {
+      cmd.trackerName = command[1] || 'default'
+      cmd.pluginName = command[2] || ''
+      cmd.methodName = command[3]
+      cmd.methodArgs = [].slice.call(queueCommand, 1)
+      if (!cmd.pluginName) {
+        if (command[1]) {
```

Comment:
When use plugin, trackerName write before period (ex, `name.pluginName:funcName`). So, swap trackerName and methodName.